### PR TITLE
Test 1

### PR DESCRIPTION
**Pull request by hamimraavi**

### CODE REVIEW

File: dyscover/tests.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
 from django.test import TestCase
 
-# Create your tests here.
+class DyscoverViewsTestCase(TestCase):
+    def test_index(self):
+        resp = self.client.get('/api/search/?q=delhi')
+        self.assertEqual(resp.status_code, 200)
```
Comment (dash1291) [inline dyscover/tests.py:9]: a test case for erroneous response too would be nice

Comment (hamimraavi) [inline dyscover/tests.py:9]: yeah..checked those too individually... i will do them at once in the next commit

File: webpage/static_dumps/dyscover.html [hunk 2]
```diff
@@ -6,33 +6,31 @@
 <script>
 function showInput(){
   var location = document.getElementById("q_id").value;
-  var result = "<ol>";
+  var BASE_URL = "http://localhost:8000/api/search/";
```
Comment (dash1291) [inline webpage/static_dumps/dyscover.html]: will this not break when you deploy your backend somewhere else?

Comment (dash1291) [inline webpage/static_dumps/dyscover.html]: or you access your frontend outside of your local machine?

Comment (hamimraavi) [inline webpage/static_dumps/dyscover.html]: so should i deploy the api on heroku?

Comment (hamimraavi) [inline webpage/static_dumps/dyscover.html]: ok.. i have used relative path now

File: webpage/static_dumps/dyscover.html [hunk 3]
```diff
@@ -6,33 +6,31 @@
...
+  var BASE_URL = "/api/search/";
   $("#display").html("Fetching results...");
-  $.getJSON("http://localhost:8000/api/search/", {q:location}, function(data, status){
-    var restaurant_list = data["Restaurants"];
-    for (var index = 0; index < restaurant_list.length; index++){
-      var restaurant_name = restaurant_list[index]["Name"];
-      var restaurant_url = restaurant_list[index]["Url"];
-      result += "<li><a href=" + restaurant_url + ">" 
-        + restaurant_name + "</a></li>";
+  $.getJSON(BASE_URL, {q:location}, function(data, status){
```
Comment (dash1291) [inline webpage/static_dumps/dyscover.html]: space before `location`

Comment (hamimraavi) [inline webpage/static_dumps/dyscover.html]: fixed

**Pull request merged**